playbooks/roles/gitcontrol/files/github_api.py

```diff
@@ -71,7 +77,8 @@ def update_branch_protection(github_api, owner, repo_name, repo):
     rules = repo[repo_name]['protection_rules']
     if isinstance(rules, str):
         rules = {repo[repo_name]['default_branch']:
-                 read_yaml_file(f'/home/zuul/src/github.com/opentelekomcloud-infra/system-config/playbooks/templates/github/{rules}.yaml')}
+            read_yaml_file(
+                f'{template_folder}/{rules}.yaml')}
```

Comment:
We need to have full path here (where SC is being checked out. Alternative is to move rules under the orgs (at least from repo POV)

Reply:
Yes, it stored above in variable template_folder = '/home/zuul/src/github.com/opentelekomcloud-infra/system-config/playbooks/templates/github'

Reply:
ah, missed that